tests/integration/tier0/user-prompt-response/main.fmf

```diff
@@ -0,0 +1,34 @@
+summary: |
+    User response is empty
```

Comment:
```suggestion
    Various user responses in interactive mode
```

Reply:
I was doing this with the inheritance in mind.
So the output of `tmt show --name user-prompt-response/empty_username_and_password` should be :
```
summary: User response is empty
Pass an empty string for username and password
```
Which IMHO looks nicer than:
```
summary: Various user responses in interactive mode
Pass an empty string for username and password

```

Reply:
Alright.  Just that it is misleading to the `/auto_attach_pool` test which has nothing to do with the user response. Either you can move it to the sanity checks folder or override the `summary` for this particular test

Reply:
Updating the summary instead of appending.
I believe the test will get removed in a near future due.